mffpy/xml_files.py

```diff
@@ -1266,6 +1294,171 @@ def get_serializable_content(self):
         return content
 
 
+class PNSSet(XML):
+    """Parser for 'pnsSet.xml' file
+
+    These files have the following structure:
+    ```
+    <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+    <PNSSet xmlns="http://www.egi.com/pnsSet_mff"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+        <name>Physio 16 set 60hz 1.0</name>
+        <ampSeries>400</ampSeries>
+        <sensors>
+            <sensor>
+                <name>ECG</name>
+                <number>0</number>
+                <unit>uV</unit>
+                <psgType>0</psgType>
+                <mapping>1</mapping>
```

Comment:
Does the number field represent the hardware channel number? What is mapping for?

Reply:
If I'm not wrong, the number field indicates the channel index that should be used to get samples from the signal file.